charts/seed-controlplane/charts/kube-apiserver/templates/network-policy.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: kube-apiserver-deny-blacklist
+  namespace:  {{ .Release.Namespace }}
+spec:
+  podSelector:
+    matchLabels:
+      app: kubernetes
+      role: apiserver
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - ipBlock:
+        # Allow all except seed networks
+        cidr: 0.0.0.0/0
+        except:
+        - {{ .Values.seedNetworks.pod }}
+        - {{ .Values.seedNetworks.node }}
+        - {{ .Values.seedNetworks.service }}
+        - {{ .Values.cloudProviderMetadataServiceIP }}/24
```

Comment:
Would maintain the range here (/24) also in the variable. We do it also for the other network ranges here.